src/status_im/contexts/wallet/data_store.cljs

```diff
@@ -212,13 +212,12 @@
 
 (defn new->old-route-path
   [new-path]
-  (let [bonder-fees (:tx-bonder-fees new-path)
-        token-fees  (+ (money/wei->ether bonder-fees)
-                       (money/wei->ether (:tx-token-fees new-path)))]
+  (let [bonder-fees (-> new-path :tx-bonder-fees money/bignumber)
+        token-fees  (-> new-path :tx-token-fees money/bignumber)]
```

Comment:
`:tx-token-fees` is the difference between `:amount-in` and the value the receiver gets ("estimated-received"), so it already include the bonder fees